exonum-java-binding-qa-service/src/main/java/com/exonum/binding/qaservice/ApiController.java

```diff
@@ -68,13 +75,41 @@
   static final String BLOCKCHAIN_HEIGHT_PATH = BLOCKCHAIN_ROOT + "/height";
   @VisibleForTesting
   static final String BLOCKCHAIN_ALL_BLOCK_HASHES_PATH = BLOCKCHAIN_ROOT + "/block";
-  private static final String BLOCK_HEIGHT_PARAM = "blockHeight";
   @VisibleForTesting
-  static final String BLOCKCHAIN_BLOCK_TRANSACTIONS_PATH = BLOCKCHAIN_ROOT + "/block/:"
-      + BLOCK_HEIGHT_PARAM + "/transactions";
+  static final String BLOCKCHAIN_BLOCKS_PATH = BLOCKCHAIN_ROOT + "/hashesToBlocks";
```

Comment:
Maybe it's time to move these constants to separate (or inner) class, `QaPaths`? 🤔